core/Tracker/Visit.php

```diff
@@ -451,6 +457,26 @@ protected function updateExistingVisit($valuesToUpdate)
         }
     }
 
+    protected function updateIdVisitorAcrossLogTables(string $idVisitor): void
+    {
+        $allLogTables = StaticContainer::get(LogTablesProvider::class)->getAllLogTables();
```

Comment:
Have we considered adding a parameter to only return log tables that has an updatable idvisitor column, or adding a new method to the provider to return those tables, to move the concern of checking which tables to work with into the provider?